lang/py/src/avro/io.py

```diff
@@ -119,6 +124,9 @@ def validate(expected_schema, datum):
     return (isinstance(datum, int) or isinstance(datum, long)
             or isinstance(datum, float))
   elif schema_type == 'fixed':
+    if hasattr(expected_schema, 'logical_type') and \
+                    expected_schema.logical_type == 'decimal':
```

Comment:
Would this be just checking if the expected_schema is an instance of `DecimalSchema`?

Also, the style guide suggests to use parenthesis than backslash. The backslash is easier to introduce errors by accident.